app/src/main-process/app-window.ts

```diff
@@ -416,6 +416,10 @@ export class AppWindow {
     return this.window.webContents.zoomFactor
   }
 
+  public setWindowZoomFactor(zoomFactor: number) {
+    return (this.window.webContents.zoomFactor = zoomFactor)
```

Comment:
No need to return anything here, right? 🤔 
```suggestion
   this.window.webContents.zoomFactor = zoomFactor
```

Reply:
Haha.. yep sry copy/pasta remnant